arbnode/tx_pre_checker.go

```diff
@@ -35,22 +35,25 @@ const TxPreCheckerStrictnessLikelyCompatible uint = 20
 const TxPreCheckerStrictnessFullValidation uint = 30
 
 type TxPreCheckerConfig struct {
-	Strictness       uint  `koanf:"strictness" reload:"hot"`
-	RequiredStateAge int64 `koanf:"required-state-age" reload:"hot"`
+	Strictness             uint  `koanf:"strictness" reload:"hot"`
+	RequiredStateAge       int64 `koanf:"required-state-age" reload:"hot"`
+	RequiredStateMaxBlocks uint  `koanf:"required-state-max-blocks" reload:"hot"`
 }
 
 type TxPreCheckerConfigFetcher func() *TxPreCheckerConfig
 
 var DefaultTxPreCheckerConfig = TxPreCheckerConfig{
-	Strictness:       TxPreCheckerStrictnessNone,
-	RequiredStateAge: 1,
+	Strictness:             TxPreCheckerStrictnessNone,
+	RequiredStateAge:       1,
+	RequiredStateMaxBlocks: 0,
```

Comment:
I realized there's also an awkward edge without MaxBlocks where the last block was produced at 10:00.99 but it's now 10:01.00 and it thinks 1 second has passed. To alleviate this, let's set the default config to RequiredStateAge to 2 but also set RequiredStateMaxBlocks to 4 (this is equivalent to a 0.75-1 second delay on networks where blocks are constantly being produced like arb1).